File: .NET Core/Embed for your organization/UserOwnsData/Models/EmbedConfig.cs
// ----------------------------------------------------------------------------
// Copyright (c) Microsoft Corporation.
// Licensed under the MIT license.
// ----------------------------------------------------------------------------

namespace UserOwnsData.Models
{
    using System.Text.Json.Serialization;

    public class EmbedConfig
    {
        [JsonPropertyName("accessToken")]
        public string AccessToken { get; set; }

        [JsonPropertyName("embedUrl")]
        public string EmbedUrl { get; set; }

        public EmbedConfig(string accessToken, string embedUrl)
        {
            this.AccessToken = accessToken;
            this.EmbedUrl = embedUrl;
        }
    }
}

File: Java/Embed for your customers/AppOwnsData/src/main/java/com/embedsample/appownsdata/models/EmbedConfig.java
// ----------------------------------------------------------------------------
// Copyright (c) Microsoft Corporation.
// Licensed under the MIT license.
// ----------------------------------------------------------------------------

package com.embedsample.appownsdata.models;

import java.util.List;

/**
 * Properties for embedding the report 
 */
public class EmbedConfig {
	public List<ReportConfig> embedReports;

	public EmbedToken embedToken;

	public String errorMessage;
}
